src/internal/components/selectable-item/index.tsx

```diff
@@ -3,11 +3,63 @@
 import React, { useLayoutEffect, useRef } from 'react';
 import clsx from 'clsx';
 
+import { getAnalyticsMetadataAttribute } from '@cloudscape-design/component-toolkit/internal/analytics-metadata';
+
 import { BaseComponentProps, getBaseProps } from '../../base-component';
 import { HighlightType } from '../options-list/utils/use-highlight-option.js';
+import { GeneratedAnalyticsMetadataSelectableItemSelect } from './analytics-metadata/interfaces';
 
+import optionAnalyticsSelectors from './../option/analytics-metadata/styles.css.js';
+import analyticsSelectors from './analytics-metadata/styles.css.js';
 import styles from './styles.css.js';
 
+export interface ItemDataAttributes {
+  'data-group-index'?: string;
+  'data-child-index'?: string;
+  'data-in-group-index'?: string;
+  'data-test-index'?: string;
+}
+
+const getAnayticsMetadata = ({
```

Comment:
typo in the naming, should be "getAnalyticsMetadata" instead

Reply:
Addressed